modules/roxotAnalyticsAdapter.js

```diff
@@ -37,12 +37,16 @@ function buildEventStack() {
 
 function send(eventType, data, sendDataType) {
   let fullUrl = url + '?publisherIds[]=' + initOptions.publisherIds.join('&publisherIds[]=') + '&host=' + window.location.hostname;
-
-  ajax(
-    fullUrl,
-    (result) => utils.logInfo('Event ' + eventType + ' sent ' + sendDataType + ' to roxot prebid analytic with result' + result),
-    JSON.stringify(data)
-  );
+  let xhr = new XMLHttpRequest();
```

Comment:
Depending on the publisher's setup, some requests are processed longer than others. But when it takes more than 3 sec to record all events, requests are canceled by the recently implemented timeout to ajax requests. As a result, we may lose some of the data. While optimizing our system and speeding up data processing, we want to make sure nothing is lost.